### PR TITLE
OSDOCS-9994: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-9994

Link to docs preview: https://73488--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes#rhcos-now-uses-rhel-9-4

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@mike-nguyen could you PTAL for QE ack? this docs PR adds a RN entry about RHCOS using RHEL 9.4 for OCP 4.16. thank you!

cc: @marrusl

File: release_notes/ocp-4-16-release-notes.adoc
Comment (bburt-rh) [inline release_notes/ocp-4-16-release-notes.adoc:60]:
```suggestion
{op-system} now uses {op-system-base-full} 9.4 packages in {product-title} {product-version}. These packages ensure that your {product-title} instances receive the latest fixes, features, enhancements, hardware support, and driver updates. As an Extended Update Support (EUS) release,{product-title} 4.14 is excluded from this change and will continue to use {op-system-base} 9.2 EUS packages for the entirety of its lifecycle.
```
- Fix typo
- Consider the edit I made to the second sentence. Not sure if I changed the meaning, but as it's currently worded it seems a bit awkward. Lmk if you have questions.

**Review by bburt-rh — APPROVED** [review]
LGTM. One typo to fix and one suggested edit to consider.

As a reminder, this PR still needs QE approval before merging.

**Comment by mike-nguyen** [comment]
Content looks good to me

File: release_notes/ocp-4-16-release-notes.adoc
Comment (jldohmann) [reply release_notes/ocp-4-16-release-notes.adoc:60]: this does sound better, thanks Brian!

**Pull request merged**
